### PR TITLE
Add scripts to automate upload

### PR DESCRIPTION
**Pull request by dave-tucker**
Once video is edited, it should be as easy as documented in the README

Signed-off-by: Dave Tucker <dave@dtucker.co.uk>

### CODE REVIEW

File: scripts/ep6.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (mestery) [inline scripts/ep6.json:1]: I don't think we want this file merged in, can you remove confirm and remove it if so?

**Review by mestery — CHANGES_REQUESTED** [review]
I think we should remove the `scripts/ep6.json` file, otherwise this is good!

**Review by mestery — APPROVED** [review]

**Comment by mestery** [comment]
@dave-tucker  this is fine to me, if you agree, either merge it or let me know and I will merge it! Thanks!

**Comment by dave-tucker** [comment]
Yeah let's merge it. I thought we could check in the episode yaml files. Why not? At least if we ever need to re-upload an episode the file is there 😉

**Pull request merged**